src/Admin/AdherentAdmin.php

```diff
@@ -440,6 +440,12 @@ protected function configureDatagridFilters(DatagridMapper $datagridMapper)
                         $qb->setParameter('committee_privileges', $privileges);
                     }
 
+                    // Deputy
+                    if (\in_array(AdherentRoleEnum::DEPUTY, $value['value'], true)) {
+                        $qb->leftJoin(sprintf('%s.managedDistrict', $alias), 'district');
```

Comment:
what about replace `leftJoin` by `innerJoin` and remove `where` condition?

Reply:
@ottaviano I agree with `innerJoin`, but we need a condition when it's a filter by role https://github.com/EnMarche/en-marche.fr/pull/2517/files#diff-d435c0b62bf9827706902379559ff9c8R421 or we should add check that `where` condition has some parts to add it or not to query.

Reply:
oh I see, keep your solution ;)